#!/usr/bin/env python
PACKAGE = "week3msg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("bool_param",bool_t,0,"a bool parameter",True)
gen.add("float_param",double_t,0,"a float parametre",.5)

exit(gen.generate(PACKAGE, "week3msg", "dynamic"))
